systemvm/debian/etc/vmware-tools/tools.conf

```diff
@@ -0,0 +1,17 @@
+[logging]
+# Turns on logging globally. It can still be disabled for each domain.
+# log = true
+
+# Disables core dumps on fatal errors; they're enabled by default.
+enableCoreDump = false
```

Comment:
Does this work with openvmtools @PaulAngus as we don't install vmware tools, unless you've changed that.

Reply:
Yes, this a bug in the Debian distribution of open-vm-tools.
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=889884